packages/safe-core-sdk/src/safeFactory/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { SafeAccountConfig } from './'
+
+export const validateSafeAccountConfig = ({ owners, threshold }: SafeAccountConfig): void => {
+  if (owners.length <= 0) throw new Error('Owner list must have at least one owner')
+  if (threshold <= 0) throw new Error('Threshold must be greater than or equal to 0')
```

Comment:
https://github.com/gnosis/safe-contracts/blob/186a21a74b327f17fc41217a927dea7064f74604/contracts/base/OwnerManager.sol#L29
```suggestion
  if (threshold <= 0) throw new Error('Threshold must be greater than or equal to 1')
```